assets/js/map.js

```diff
@@ -144,7 +144,7 @@ const MapBase = {
 
     MapBase.map.doubleClickZoom[Settings.isDoubleClickZoomEnabled ? 'enable' : 'disable']();
 
-    const southWest = L.latLng(-160, -120),
+    const southWest = L.latLng(-160, -210),
```

Comment:
-210 is definitely not a correct value.
on 2nd zoom lvl you can move the map out of the screen

Reply:
-170 is reasonable value i think, you can't move the map out of the screen on high dpi screens, and New Austin is not hidden under fme events on low dpi screens

Reply:
I could give you a screen shot... But nevertheless I removed that commit so this merge request is only the file split now.